Move startx auth files in /tmp so they are removed on reboot.
http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=357736
The trap patch didn't seem to work on reboot.
---
 startx.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: xinit/startx.cpp
===================================================================
--- xinit.orig/startx.cpp	2009-12-06 23:26:32.000000000 +0100
+++ xinit/startx.cpp	2009-12-06 23:26:37.000000000 +0100
@@ -271,7 +271,7 @@
     dummy=0
 
     XCOMM create a file with auth information for the server. ':0' is a dummy.
-    xserverauthfile=$HOME/.serverauth.$$
+    xserverauthfile=`mktemp --tmpdir serverauth.XXXXXXXXXX`
     trap "rm -f '$xserverauthfile'" HUP INT QUIT ILL TRAP KILL BUS TERM
     xauth -q -f "$xserverauthfile" << EOF
 add :$dummy . $mcookie
